### ISSUE TITLE
Grappl.io offline

### ISSUE THREAD

**Issue by BenCat07** [ellenhebert/Grappl#22]
The website is ofline. Fix it

**Comment by ellenhebert** [comment]
unlikely it will be fixed anytime soon. it's down intentionally because there's a lot of issues that need to be worked out with it

**Comment by BenCat07** [comment]
Okay thanks for replying

Am 23.08.2016 19:11 schrieb "Ellen Hebert" <notifications@github.com>:

**Comment by starwarswii** [comment]
@ellenhebert Would you be able to put up a simple page explaining that temporarily? Otherwise Nobody would know this without coming to the GitHub.

**Comment by BenCat07** [comment]
Yeah, that should be better for people like me who don't look at everything
on githubb

2016-09-21 5:13 GMT+02:00 starwarswii <notifications@github.com>:
